pycaret/time_series/forecasting/oop.py

```diff
@@ -3055,6 +3056,16 @@ def _plot_model_get_train_test_split_data(
 
         return data, data_label
 
+    @staticmethod
+    def plot_model_check_display_format_(display_format: Optional[str]):
```

Comment:
Can you add the doctrting in numpy format with the argument descripton?

Reply:
As for now, I just copy-pasted the docstring from this code-part: https://github.com/pycaret/pycaret/blob/98cbf9451f1c20cc701b406f0b9b72d21f4ec2a6/pycaret/internal/pycaret_experiment/tabular_experiment.py#L361-L364

We might then also update that docstring?

Reply:
in https://github.com/pycaret/pycaret/pull/2671/commits/8c0dc92baede6b699a3fc0b31f4cefaa28f3fdf6 I updated the docstring in `oop.py`